portal/src/graphql/adminapi/UserDetailCommandBar.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useContext, useMemo } from "react";
+import cn from "classnames";
+import { CommandBar, ICommandBarItemProps } from "@fluentui/react";
+
+import { Context } from "@oursky/react-messageformat";
+
+import styles from "./UserDetailCommandBar.module.scss";
+
+interface UserDetailCommandBarProps {
+  className?: string;
+}
+
+const UserDetailCommandBar: React.FC<UserDetailCommandBarProps> = function UserDetailCommandBar(
+  props: UserDetailCommandBarProps
+) {
+  const { className } = props;
+  const { renderToString } = useContext(Context);
+
+  const commandBarItems: ICommandBarItemProps[] = useMemo(
+    () => [
+      {
+        key: "remove",
+        text: renderToString("remove"),
+        iconProps: { iconName: "Delete" },
+      },
+
+      {
+        key: "loginAsUser",
+        text: renderToString("UserDetails.command-bar.login-as-user"),
+        iconProps: { iconName: "FollowUser" },
+      },
+
+      {
+        key: "invalidateSessions",
+        text: renderToString("UserDetails.command-bar.invalidate-sessions"),
+        iconProps: {
+          iconName: "CircleAddition",
```

Comment:
It seems this button is gone in the design.

Reply:
It should be moved to session screen, let's remove it for now and add it to session screen when we update that screen